server/app/views/style/StyleUtils.java

```diff
@@ -113,4 +113,13 @@ public static String responsive2XLarge(ImmutableList<String> styles) {
   public static String joinStyles(String... styles) {
     return String.join(" ", styles);
   }
+
+  public static String removeStyles(String style, String... stylesToRemove) {
+    return Stream.of(stylesToRemove)
+        .reduce(
+            style,
+            (acc, styleToRemove) -> acc.replace(styleToRemove, ""),
+            (acc1, acc2) -> acc1 + acc2)
```

Comment:
I don't think you need to pass the last function. Having only accumulator function is enough.

Reply:
Done. Yeah that code was actually written by Github Copilot